Makefile

```diff
@@ -27,7 +27,7 @@ docker_all: docker_debs docker_el7_rpms docker_el6_rpms
 
 package: docker_all
 	sudo chmod 777 build
-	cp docs/release_readme.rst RELEASE_README.rst
+	mv release_readme.rst RELEASE_README.rst
```

Comment:
The `cp` actually failed, which is why I changed it to `mv`. I think we need to go over this together.